internal/driver/driver.go

```diff
@@ -634,13 +588,13 @@ func (d *Driver) AddDevice(deviceName string, protocols protocolMap, adminState
 // update the address, and attempt to reconnect at the new address and port.
 // If the address is the same, nothing happens.
 func (d *Driver) UpdateDevice(deviceName string, protocols protocolMap, adminState contract.AdminState) (err error) {
-	d.lc.Debug(fmt.Sprintf("Updating device: %s protocols: %v adminState: %v",
+	d.lc.Debug(fmt.Sprintf("Updating device: %s protocols: %+v adminState: %+v",
```

Comment:
also, `adminState` is technically a string type
```suggestion
	d.lc.Debug(fmt.Sprintf("Updating device: %s protocols: %v adminState: %v",
```

Reply:
removed +